db/migrate/20161107112025_remove_available_currencies_column_from_communities.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveAvailableCurrenciesColumnFromCommunities < ActiveRecord::Migration
+  def up
+    remove_column :communities, :available_currencies
+  end
```

Comment:
Rollback migration is missing.

But instead of adding `down` migration, you can just rename the `up` to `change` and add some information to `remove_column`:

```
def up
  remove_column :communities, :available_currencies, :string, limit: 3, after: ...
end
```